.github/workflows/ci.yml

```
@@ -28,21 +31,42 @@ jobs:
           else
             GIT_REF="${{ github.ref }}"
           fi
-          echo "GIT_REF=${GIT_REF}" >>"$GITHUB_ENV"
+          echo "git_ref=${GIT_REF}" >>"$GITHUB_OUTPUT"
+
+  docker_smoketests:
+    name: Smoketests
+    needs: find_ref
+    strategy:
+      matrix:
+        include:
+          - { runner: spacetimedb-runner, smoketest_args: --docker }
+          - { runner: windows-latest, smoketest_args: --no-build-cli }
+        runner: [spacetimedb-runner, windows-latest]
+    runs-on: ${{ matrix.runner }}
+    steps:
       - name: Checkout sources
         uses: actions/checkout@v4
         with:
-          ref: ${{ env.GIT_REF }}
+          ref: ${{ needs.find_ref.outputs.git_ref }}
       - uses: dsherret/rust-toolchain-file@v1
-      - uses: actions/setup-dotnet@v3
+      - uses: actions/setup-dotnet@v4
         with:
           global-json-file: modules/global.json
       - name: Start containers
+        if: runner.os == 'Linux'
         run: docker compose up -d
+      - name: Start database
```

So that it's easier to understand when looking at the CI output:
```suggestion
      - name: Start database (Windows)
```